ExampleTabbedApp/AppDelegate.m

```diff
@@ -0,0 +1,82 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "AppDelegate.h"
+#import "AppTabbedViewController.h"
+
+#import <MaterialComponents/MaterialFlexibleHeader.h>
+#import <MaterialComponents/MaterialNavigationBar.h>
+#import <MaterialComponents/MaterialTabs.h>
+
+@interface AppDelegate ()
+
+@end
+
+@implementation AppDelegate
+
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+  self.window = [[UIWindow alloc] initWithFrame:[UIScreen mainScreen].bounds];
+  AppTabbedViewController *rootViewController =
+      [[AppTabbedViewController alloc] initWithNibName:nil bundle:nil];
+  self.window.rootViewController = rootViewController;
+  [self.window makeKeyAndVisible];
+  [self configureUIAppearanceSettings];
+  return YES;
+}
+
+- (void)configureUIAppearanceSettings {
+  // AppBar
```

Comment:
Since this is template code, it *may* make sense to move this into application:didFinishLaunching: rather than declaring a new method.  My thinking is that all of the other methods are delegate methods.  Something to consider.

Since this is a 'template', consider adding a comments underneath our code:
// Use the UIAppearance Proxy to set the colors of your controls